description.md

```diff
@@ -0,0 +1,7 @@
+#Zoom-bot
+The Zoom-bot is an interactive navigation and maintence interface that could be implemented on subways and trains all over the world to improve riders experience.
+The Zoom-bot is an interface to be placed on trains and subways. It should assist users by cleaning the train and giving them information including directions and air quality. The Zoom-bot will also keep track of train cleanliness and alert maintenance workers of problems. Subway employees will be using the interface to track trains’ cleanliness and location. 
+#Clinique Virtual Reality
+Clinique Virtual Reality is a way to experience personalized cosmetic advice from any location, at any time. It allows the user to scan their face, view the problem areas, and recommends products and solutions. 
+#UXPA
+As the founding treasurer of UXPA, Penn State Chapter, I have put in a lot of work to get the club started. UXPA strives to give UX students oportunities to gain expreience outside of the classroom. We haven't currently finished any projects, but we are well on our way.
```

Comment:
The first two sound pretty good, but the last one does not quite measure up. I would not say anything about projects if you have not done any, but I would replace "put in a lot of work" with a list of your specific responsiblities or things you have done. Maybe consider flipping the first two sentances.